Formula/apbspdb2pqr.rb

```diff
@@ -0,0 +1,56 @@
+class Apbspdb2pqr < Formula
+  include Language::Python::Virtualenv
+  # cite Jurrus_2018: "https://doi.org/10.1002/pro.3280"
+  # cite Baker_2001: "https://doi.org/10.1073/pnas.181342398"
+  desc "Electrostatic and solvation properties for complex molecules"
+  homepage "https://www.poissonboltzmann.org/"
+  # pull from git tag to get submodules
+  url "https://github.com/Electrostatics/apbs-pdb2pqr.git",
+      :tag      => "apbs-1.5",
+      :revision => "aa353941cfadc09ccd113075d261a427864c2979"
+  head "https://github.com/Electrostatics/apbs-pdb2pqr.git"
+
```

Comment:
I think this might need a `version "1.5"` directive too.

Reply:
With `version "1.5"`,  I have got an error:
```
apbspdb2pqr:
  * Stable: version 1.5 is redundant with version scanned from URL
Error: 1 problem in 1 formula detected
```
Does it mean that the directive `version "1.5"` is not required?

Reply:
Yes, `version "1.5"` is not required then.